src/stream/src/executor/global_simple_agg.rs

```diff
@@ -173,9 +183,17 @@ impl<S: StateStore> GlobalSimpleAggExecutor<S> {
             .zip_eq(state_tables.iter_mut())
         {
             let vis_map = agg_call_filter_res(agg_call, &columns, visibility.as_ref(), capacity)?;
-            agg_state
-                .apply_batch(&ops, vis_map.as_ref(), data, epoch, state_table)
-                .await?;
+            if agg_call.kind == AggKind::StringAgg {
+                let chunk_cols = columns.iter().map(|col| col.array_ref()).collect_vec();
+                agg_state
+                    .apply_batch(&ops, vis_map.as_ref(), &chunk_cols, epoch, state_table)
+                    .await?;
+            } else {
+                // TODO(yuchao): Pass all the columns to agg states' apply_batch for other agg calls
```

Comment:
FYI, in other words, it will be `apply_batch(stream_chunk, epoch, state_table)`

Reply:
yep, we can rename the function and/or parameters/arguments in later PR